woodwork/tests/utils/test_read_file.py

```diff
@@ -82,7 +82,7 @@ def test_read_file_validation_control(
             "sample.csv",
             ("to_csv", {"index": False}),
             {
-                "nrows": 2,
+                "nrows": 3,
```

Comment:
changing to 3 rows because otherwise, `id` has values `'0', '1'`, which gets inferred as boolean